enchant.lic

```diff
@@ -59,10 +59,6 @@ class Enchant
     @equipment_manager = EquipmentManager.new
     @equipment_manager.empty_hands
 
-    if @use_own_brazier
```

Comment:
💡 Can we remove the `DRCT` references from this script now?

Reply:
Done!